tests/sles4sap/sapconf.pm

```diff
@@ -79,7 +79,7 @@ sub run_developers_tests {
             record_soft_failure "bsc#$1";
         }
         else {
-            record_soft_failure $summary;
+            record_info $summary. result => 'fail';
```

Comment:
That dot `.` should be a colon `,`, right? Or I am going blind and it is already a colon?

Also, the test shell script is now referencing 3 tests that are not tied to a bsc (example in: https://openqa.suse.de/tests/1750092#step/sapconf/111), one of those is tied to a fate, so I'd like to add this block before the else:

```
elsif ($summary =~ /Test #fate([0-9]+)/) { record_soft_failure "fate#$1"; }
```

Or better yet, re-write the regexp on the if as so:

```
if ($summary =~ /Test #(bsc|fate)([0-9]+)/) {
    record_soft_failure $1 . "#" . $2;
}
```